src/sslyze_csv2mongo.py

```diff
@@ -6,6 +6,7 @@
 import csv
 from datetime import datetime, time, timezone
 import os
+from zoneinfo import ZoneInfo
```

Comment:
Thoughts on removing the import of `datetime.timezone` and replacing the use of `timezone.utc` with `ZoneInfo("UTC")` just to consolidate? I'm not sure if we want to deviate from how the other (pshtt/trustymail) scripts get UTC time zone information or want to convert them in the same way. I'm fine either way on this one but figured it was worth mentioning since we're now using another library for time zone information as well.

Reply:
I prefer using time zone information from `timezone` where possible (which I think is only for UTC), and reaching into `ZoneInfo` as necessary.

Reply:
Ok. It's only UTC because `datetime.timezone.utc` is essentially `datetime.timezone(datetime.timedelta(0))` since the `datetime` package has no actual understanding of time zones.

Reply:
Can we resolve this conversation?

Reply:
Ugh.  See commit b300f58.

Reply:
Ah, sorry, I hadn't meant to seem insistent on making this change. Uhmm thank you anyway?